src/components/statistics/Statistics.tsx

```diff
@@ -11,22 +11,41 @@ function Statistics() {
   const [deathsDelta, setDeathsDelta] = useState(0);
   const [allVaccinations, setAllVaccinations] = useState(0);
   const [vaccinationsDelta, setVaccinationsDelta] = useState(0);
+  const [sevenDayIncident, setSevenDayIncident] = useState(0);
 
   Api.makeApiRequestToRKIData(UrlConfig.urlAllInfections).then(result => setAllInfections(result));
   Api.makeApiRequestToRKIData(UrlConfig.urlInfectionsDelta).then(result => setInfectionsDelta(result));
   Api.makeApiRequestToRKIData(UrlConfig.urlAllDeaths).then(result => setAllDeaths(result));
   Api.makeApiRequestToRKIData(UrlConfig.urlDeathsDelta).then(result => setDeathsDelta(result));
   Api.makeApiRequestToRKIData(UrlConfig.urlAllVaccinations).then(result => setAllVaccinations(result));
   Api.makeApiRequestToRKIData(UrlConfig.urlAllVaccinationsDelta).then(result => setVaccinationsDelta(result));
+  Api.makeIncidentRequest(UrlConfig.urlIncident).then(result => setSevenDayIncident(result));
+
   const colorOfElement = infectionsDelta > 10000 ? "#c42929" : "black";
+  
+  let colorOfInfections = "black";
+  if(sevenDayIncident > 100){
+    colorOfInfections =  "red" //"#c42929"
+  }
+  else if(sevenDayIncident < 50){
+    colorOfInfections =  "green" //"#43c429"
+  }
+  else{
+    colorOfInfections = "yellow" //"#e0de4c"
+  }
 
   return (
       <div className="Statistics" id="Statistics">
         <Typography className="Header">Aktuelle Zahlen zur Corona-Pandemie in Deutschland</Typography>
         <Typography className="HeaderSmall">Aktuelle Zahlen </Typography>
 
         <Typography className="AllInfectionsHeader">Alle Infektionen:</Typography>
-        <Typography className="AllInfections">{Formatter.formatNumber(allInfections)}</Typography>
+        <Typography 
+            className="AllInfections"
+            style={{ color: colorOfInfections}}
```

Comment:
hier wird die farbe dann gleich "red" oder "green" etc. gesetzt, und nicht das, was du oben in deinen kommentaren stehen hast